### ISSUE TITLE
sealer login not enter username  password

### ISSUE THREAD

**Issue by cuisongliu** [alibaba/sealer#38]
#### What happened:

1.  enter `sealer login registry-vpc.cn-hongkong.aliyuncs.com` no enter username and password auto login tips success
3. cat ~/.docker/config.json 
  ```
     {
	"auths": {
		"registry-vpc.cn-hongkong.aliyuncs.com": {
			"auth": "Og=="
		},
		"registry-vpc.cn-hongkong.aliyuncs.com/sealyun/kubernetes-amd64": {
			"auth": "Og=="
		},
		"registry.cn-beijing.aliyuncs.com": {
			"auth": "xxxxxxx"
		}
	}
}
  ```
#### Environment:
- sealer version (use `sealer version`):
- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

**Comment by bxy4543** [comment]
fix login not enter username password #65

**Comment by cuisongliu** [comment]
```
root@sealer-node0:~# sealer pull registry.cn-qingdao.aliyuncs.com/sealer-io/cloudrootfs:v1.19.9-alpha.3
2021-05-17 19:25:19 [WARN] failed to get auth info, err: open /root/.docker/config.json: no such file or directory
2021-05-17 19:25:19 [EROR] Get "http:///v2/": http: no Host in request URL
```
